topi/python/topi/__init__.py

```diff
@@ -16,6 +16,7 @@
 from .reduction import *
 from .transform import *
 from .broadcast import *
+from .nn import *
```

Comment:
We do not want to expose nn namespace to root, instead create another folder called vision